docs/source/prgpatterns/lockstep.rst

```diff
@@ -37,10 +37,10 @@ pmacc helpers
    :project: PIConGPU
 
 .. doxygenstruct:: pmacc::lockstep::Idx
-  :project: PIConGPU
+   :project: PIConGPU
 
-.. doxygenstruct:: pmacc::lockstep::Worker
-  :project: PIConGPU
+.. doxygenclass:: pmacc::lockstep::Worker
+   :project: PIConGPU
```

Comment:
Not sure if that misalignment was important, but consistency doesn't hurt anyways.